### DISCUSSION TITLE
Is it possible to merging 3D models into one model and checking them in right postion ?

### DISCUSSION THREAD

**Discussion by ubozorboyev** [sceneview/sceneview#575] [General]
I need to do insturction for lab equipments to how to use them ? 
If they are moving I need to show direction. 
If they are in the wrong position, I have to show the right position.

**Comment by ThomasGorisse** [comment]
In v4.0.0-rc.1, you can compose multiple models as sibling nodes with relative positions:

```kotlin
SceneView {
    val model1 = rememberModelInstance(modelLoader, "models/part1.glb")
    val model2 = rememberModelInstance(modelLoader, "models/part2.glb")
    
    if (model1 != null) {
        ModelNode(modelInstance = model1, position = Position(0f, 0f, 0f))
    }
    if (model2 != null) {
        ModelNode(modelInstance = model2, position = Position(1f, 0f, 0f))
    }
}
```

SceneView doesn't merge meshes at runtime, but you can position multiple models as a group. For true mesh merging, do it in Blender or your 3D tool before exporting as a single GLB.
